api/lib_test.go

```diff
@@ -197,12 +197,11 @@ func TestGetMetrics(t *testing.T) {
 
 	// GetMetrics 3
 	metrics, err = GetMetrics(cache)
-	require.NoError(t, err)
-	assert.Equal(t, &types.Metrics{
-		HitsFsCache:         1,
-		ElementsMemoryCache: 1,
-		SizeMemoryCache:     5665691,
-	}, metrics)
+	assert.NoError(t, err)
+	require.Equal(t, uint32(0), metrics.HitsMemoryCache)
+	require.Equal(t, uint32(1), metrics.HitsFsCache)
+	require.Equal(t, uint64(1), metrics.ElementsMemoryCache)
+	require.InEpsilon(t, 5602873, metrics.SizeMemoryCache, 0.18)
```

Comment:
Where the value comes from?

Reply:
The upstream value was used.
https://github.com/CosmWasm/wasmvm/blob/9d865278deafd7d012f2347ca87eedc27586dcfa/internal/api/lib_test.go#L217

Reply:
How about copying value from our main repository?

Reply:
Fixed.